cmd/lakefs/cmd/run.go

```diff
@@ -203,6 +203,11 @@ const runBanner = `
 
 func printWelcome(w io.Writer) {
 	_, _ = fmt.Fprint(w, runBanner)
+	_, _ = fmt.Fprintf(w, "Version %s\n", config.Version)
+	catalogerType := cfg.GetCatalogerType()
+	if len(catalogerType) > 0 {
```

Comment:
cataloger type is implemented to have a default, but that default is *not* returned by `GetCatalogerType` (I assume we didn't do the whole config dance because a temporary config flag).  I think we should print the default if there is no cataloger type -- otherwise it will be hard to tell what's *really* running, just when it matters most!